source/sdk/lang/equalities.ooc

```diff
@@ -0,0 +1,47 @@
+// The following functions were moved from HashMap.ooc
```

Comment:
The only other examples I've been able to find (except things that were obviously not code) are either wrapper files (files that specify functions in other libraries, and I'm not sure it's correct in that case), shaders (not sure why, but I remember we decided to do that), data files (large, dumb arrays and no actual functions or anything). @simonmika, input please...